test/test.py

```diff
@@ -0,0 +1,168 @@
+import logging
+import mock
+import yaml
+import test_mock
+import unittest
+from unittest import TestCase
+from reviewrot.githubstack import GithubService
+from reviewrot.gitlabstack import GitlabService
+from reviewrot.pagurestack import PagureService
+from github.GithubException import BadCredentialsException
+from gitlab.exceptions import GitlabConnectionError
+
+# Disable logging to avoid messing up test output
+logging.disable(logging.CRITICAL)
+
+
+class GithubTest(TestCase):
+    def test_object_create(self):
+        isinstance_ = isinstance(GithubService(), GithubService)
+        self.assertEqual(isinstance_, True)
+
+    def test_request_review_token(self):
+        with open('test_githubtest.yaml', 'r') as f:
+            config = yaml.load(f)
+        github = GithubService()
+        with self.assertRaises(BadCredentialsException) as context:
+            github.request_reviews(user_name=config['user_name'],
+                                   token=config['token'])
+        self.assertTrue('Bad credentials' in str(context.exception))
+
+    @mock.patch('github.Github.get_user', side_effect=test_mock.mock_get_user)
+    def test_request_reviews_get_user(self, mock_get_user):
+        with open('test_githubtest.yaml', 'r') as f:
+            config = yaml.load(f)
+        with self.assertRaises(Exception) as context:
+            GithubService().request_reviews(user_name=config['user_name'],
+                                            token=config['token'])
+        msg = 'Invalid username/organizaton: %s' % config['user_name']
+        self.assertTrue(msg in str(context.exception))
+
+    @mock.patch('github.Github.get_user', side_effect=test_mock.mock_get_user_)
+    @mock.patch('reviewrot.githubstack.GithubService.get_reviews',
+                side_effect=test_mock.mock_github_get_reviews)
+    def test_request_reviews_with_repo(self, mock_get_user_,
+                                       mock_github_get_reviews):
+        with open('test_githubtest.yaml', 'r') as f:
+            config = yaml.load(f)
+        res = GithubService().request_reviews(user_name=config['user_name'],
+                                              token=config['token'],
+                                              repo_name=config['repo_name'])
+        msg = config['msg']
+        result = []
+        result.append(msg)
+        self.assertTrue(result, res)
```

Comment:
Two things here (and they apply across a few of the other tests too):

1) Do you mean to use `assertTrue`? The value of `result` always comes from `config` and will always be "truthy". This assertion will always pass because of this. I'm guessing you actually meant to use `assertEqual`. Is this the case or did I interpret it wrong?

2) Rather than these 4 lines, you can do it all in one elegant call.

```python
self.assertEqual([config['msg']], res)
```